### PR TITLE
More translation and re-wording

### PR DESCRIPTION
**Pull request by cquest**

### CODE REVIEW

File: lib/l10n/app_fr.arb [hunk 1]
```diff
@@ -26,10 +26,10 @@
   "newSequenceNameField_placeholder": "Saisissez le nom de la nouvelle séquence",
   "newSequenceSendButton": "Envoyer",
 
-  "osmLicence": "Photos dans le Monde entier publiées en open data (licence CC-BY-SA 4.0)",
+  "osmLicence": "Photos dans le Monde entier publiées  sous licence CC-BY-SA 4.0",
```
Comment (trouyer) [inline lib/l10n/app_fr.arb:29]:
```suggestion
  "osmLicence": "Photos dans le Monde entier publiées sous licence CC-BY-SA 4.0",
```

File: lib/l10n/app_en.arb [hunk 2]
```diff
@@ -31,10 +31,10 @@
   "gpsIsDisableContent": "Please activate your GPS, the application cannot work without it",
   "gpsIsDisableTitle": "GPS disabled",
 
-  "osmLicence": "Photos around the world published in open data (CC-BY-SA 4.0 license)",
+  "osmLicence": "Photos around the world published under CC-BY-SA 4.0 license",
   "instanceOsmTitle": "OpenStreetMap France",
-  "ignLicence": "Photos in France published in open data (etalab-2.0 license)",
-  "instanceIgnTitle": "Institut Géographique Nationale",
+  "ignLicence": "Photos in France published under Licence ouvert 2.0 (CC-BY)",
```
Comment (trouyer) [inline lib/l10n/app_en.arb:36]:
```suggestion
  "ignLicence": "Photos in France published under Licence ouverte 2.0 (CC-BY)",
```

**Review by trouyer — COMMENTED** [review]
2 suggestions that fix typo

**Comment by OtterWays** [comment]
These strings are no longer up to date.

**Pull request closed without merging**